preventsleep/linux.go

```diff
@@ -11,18 +11,21 @@ import (
 )
 
 const (
-	inhibitWhat      = "sleep"
-	inhibitWhy       = "Backup and/or restic repository maintenance"
-	inhibitMode      = "block"
-	permissionDenied = "Permission denied"
+	inhibitWhatSleep    = "idle:sleep"
+	inhibitWhatShutdown = "idle:sleep:shutdown"
```

Comment:
Considered making this default for when `prevent-sleep` but wasn't sure that was a good assumption

Reply:
I added this because I often shut down without realising my backups are running, this will wait for them to finish first

Reply:
Opted to make this default rather than trying to implement for all platforms. Not sure what the default is on Windows/macOS but I think it's reasonable to assume this might be wanted on Linux. Happy to change though.

Reply:
I think I will leave this out for now, it seems that it gets overridden when I shut down graphically anyway, compared to sleep where it prompts you if you're sure

Reply:
Ok. You may add it as a separate PR. Think it is a good contribution and I liked first approach with an extra setting next to `PreventSleep`. However if it can't be done in a stable way it may be better to leave it or understand and document / log the limitations.